packages/shared/lib/services/sync/config/deploy.service.ts

```diff
@@ -835,6 +844,31 @@ async function compileDeployInfo({
     };
 }
 
+async function markSyncConfigAsInactive(oldSyncConfigId: number): Promise<void> {
```

Comment:
I would just rename for explicitness, this method switch active state from one sync_config id to another